src/main/java/no/nav/familie/inntektsmelding/refusjonomsorgsdagerarbeidsgiver/rest/RefusjonOmsorgsdagerArbeidsgiverRest.java

```diff
@@ -61,4 +72,19 @@ public RefusjonOmsorgsdagerArbeidsgiverRest(ArbeidstakerTjeneste arbeidstakerTje
         }
         return Response.ok(dto).build();
     }
+
+    @GET
+    @Path(INNLOGGET_BRUKER)
+    @Produces(MediaType.APPLICATION_JSON + ";charset=utf-8")
+    @Operation(description = "Henter opplysninger om innlogget bruker.", tags = "imdialog")
+    @Tilgangskontrollert
+    public Response hentInnloggetBruker(
+        @Parameter(description = "Hvilken ytelse den innloggete brukeren skal sende inn inntektsmelding for")
+        @NotNull @QueryParam("ytelseType") Ytelsetype ytelseType,
+        @Parameter(description = "Hvilken organisasjon den innloggete brukeren skal sende inn inntektsmelding for")
+        @NotNull @Pattern(regexp = "^\\d{9}$") @QueryParam("organisasjonsnummer") String organisasjonsnummer
```

Comment:
Burde jeg lage en egen Organisasjonsnummer type? Så det var en OrganisasjonsnummerDto, men den var for JSON-felt. Tenker det kanskje er vel så greit at man bare gjør det enkelt og greit på denne måten også.